crates/biome_cli/tests/snapshots/main_cases_diagnostics/max_diagnostics_verbose.snap

```diff
@@ -127,9 +115,41 @@ src/folder_0/package-lock.json project  VERBOSE  ━━━━━━━━━━
     i You can hide this diagnostic by using --diagnostic-level=warn to increase the diagnostic level shown by CLI.
     
 
+```
+
+```block
+src/file.js format ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × File content differs from formatting output
+  
+    1   │ - ··statement(··)··
+      1 │ + statement();
+      2 │ + 
+  
+
 ```
 
 ```block
 Checked 1 file in <TIME>. No fixes applied.
 Found 1 error.
 ```
+
+```block
+ VERBOSE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  i List of files that were evaluated:
+  
+  - src/file.js
+  
+
+```
+
+```block
+ VERBOSE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  i List of files that were fixed:
```

Comment:
We could simplify the wording to `List of fixed files:` or even `Fixed files:`/`Fixed:`